CMakeLists.txt

```diff
@@ -0,0 +1,295 @@
+cmake_minimum_required(VERSION 3.7 FATAL_ERROR)
```

Comment:
I'm pretty sure I'm not using any new features, but I don't know how far back it will work, but the documentation is perplexing:

> The FATAL_ERROR option is accepted but ignored by CMake 2.6 and higher. It should be specified so CMake versions 2.4 and lower fail with an error instead of just a warning.

Maybe just remove the FATAL_ERROR anyways, since it **_is_** threatening